sites/svelte.dev/src/routes/+page.svelte

```diff
@@ -22,7 +22,8 @@
 
 <div class="blurb-shifter">
 	<Blurb>
-		<div slot="one">
+		<!-- svelte-ignore a11y-no-noninteractive-tabindex -->
+		<div slot="one" id="main" tabindex="0">
```

Comment:
Why are we setting `tabindex` here? It's not needed and may actually be detrimental, since you're making something focusable that is not interactive (hence the warning).

We also shouldn't be adding `id="main"` here -- we already have a root `<main id="main">` wrapping the whole page.

Actually, looking at the site preview, it seems like the `main` element is in the wrong place. It's currently wrapping all the content on the site, including the nav bar, but it should _only_ include the main content. This way the skip link allows users to skip past the repeated content on each page (the nav bar) and jump right to the main content.

Markup should look something like this:

```html
<a href="#main">Skip to main content</a>
<nav>Nav stuff common to all pages</nav>
<main>
<h1>Main page heading</h1>
<p>Whatever content is on the page</p>
</main>
<footer>
Footer stuff that is on all the pages
</footer>
```

Reply:
Hm, on the current svelte.dev I see `<nav>` outside of `<main>` while this PR preview has `<nav>` inside of `<main>`. So I think it is a regression from the current site. Putting nav inside of main means the skip link is effectively useless.

<img width="527" alt="image" src="https://github.com/sveltejs/svelte/assets/4992896/ea1cce33-123c-4800-b42d-0e2edcb57274">

You're right that `<footer>` is inside of `<main>` on both versions of the site, but that's less of an issue. I'd be okay with leaving that for now.

Reply:
In my own local copy of site-kit, I moved Nav outside of main. I also got a question: What do we do about the docs page. The keyboard never goes to the left sidebar, which I can fix. But how does Skip Nav works in that case? Do we have two skip navs?

Reply:
Ah, good callout. I think two skip navs would be fine in that case, i.e. an additional skip link just on documentation that jumps past the table of contents to the actual doc content.